CONTRIBUTING.rst

```diff
@@ -71,7 +71,8 @@ Commit Guidelines
 
 - Prefix each commit's titles with the module name, followed by a colon
   and a space (unless modifying a file in the base directory).  After
-  that, the first word should be capitalized.
+  that, the first word should be capitalized and the rest words should
```

Comment:
that, the first word should be capitalized and the rest of the words should
```suggestion
  that, the first word should be capitalized and the rest of the words should
```

Reply:
Fixed and wrapped at 72-column in the file.